examples/wheel/BUILD.bazel

```diff
@@ -16,15 +16,28 @@ load("@bazel_skylib//rules:build_test.bzl", "build_test")
 load("//examples/wheel/private:wheel_utils.bzl", "directory_writer", "make_variable_tags")
 load("//python:defs.bzl", "py_library", "py_test")
 load("//python:packaging.bzl", "py_package", "py_wheel")
+load("//python:pip.bzl", "compile_pip_requirements")
 load("//python:versions.bzl", "gen_python_config_settings")
 
 package(default_visibility = ["//visibility:public"])
 
 licenses(["notice"])  # Apache 2.0
 
+compile_pip_requirements(
+    name = "requirements",
+    target_compatible_with = select({
```

Comment:
nit: can be a tad shorter like https://github.com/aspect-build/bazel-lib/blob/035473873a20135d24f7697426b613731cf1301f/lib/private/docs.bzl#L20

```
        "@platforms//os:windows": ["@platforms//:incompatible"],
        "//conditions:default": [],
```